### PR TITLE
Fix the links in scaling-deployments.md and scaling-jobs.md

### PR DESCRIPTION
**Pull request by Shubham82**
Signed-off-by: Shubham82 <shubham.kuchhal@india.nec.com>

Updated the links  for  scaledobject_types.go and scaledjob_types.go in  scaling-deployments.md and scaling-jobs.md in  keda 1.4, 1.5, 2.0 and 2.1

### CODE REVIEW

**Comment by Shubham82** [comment]
@zroubalik  
Could you please take a look.
Thanks.

File: content/docs/1.4/concepts/scaling-jobs.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For example, if you wanted to use KEDA to run a job for each message that lands
 
 This specification describes the `ScaledObject` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/master/pkg/apis/keda/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```
Comment (tomkerkhove) [inline content/docs/1.4/concepts/scaling-jobs.md:22]: Let's use the exact tags to be complete:
```suggestion
[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1.4.0/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```

File: content/docs/2.0/concepts/scaling-jobs.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ For example, if you wanted to use KEDA to run a job for each message that lands
 
 This specification describes the `ScaledJob` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
 
-[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/master/api/v1alpha1/scaledjob_types.go)
+[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/main/api/v1alpha1/scaledjob_types.go)
```
Comment (tomkerkhove) [inline content/docs/2.0/concepts/scaling-jobs.md:23]: Let's use the exact tags to be complete:
```suggestion
[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/v2.0.0/api/v1alpha1/scaledjob_types.go)
```

File: content/docs/2.0/concepts/scaling-deployments.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ The only constraint is that the target `Custom Resource` must define `/scale` [s
 
 This specification describes the `ScaledObject` Custom Resource definition which is used to define how KEDA should scale your application and what the triggers are. The `.spec.ScaleTargetRef` section holds the reference to the target resource, ie. `Deployment`, `StatefulSet` or `Custom Resource`. 
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/master/api/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/main/api/v1alpha1/scaledobject_types.go)
```
Comment (tomkerkhove) [inline content/docs/2.0/concepts/scaling-deployments.md:34]: Let's use the exact tags to be complete:
```suggestion
[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v2.0.0/api/v1alpha1/scaledobject_types.go)
```

File: content/docs/1.5/concepts/scaling-jobs.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ For example, if you wanted to use KEDA to run a job for each message that lands
 
 This specification describes the `ScaledObject` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/master/pkg/apis/keda/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```
Comment (tomkerkhove) [inline content/docs/1.5/concepts/scaling-jobs.md:22]: Let's use the exact tags to be complete:
```suggestion
[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1.5.0/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```

File: content/docs/1.5/concepts/scaling-deployments.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ For example, if you wanted to use KEDA with an Apache Kafka topic as event sourc
 
 This specification describes the `ScaledObject` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/master/pkg/apis/keda/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```
Comment (tomkerkhove) [inline content/docs/1.5/concepts/scaling-deployments.md:26]: Let's use the exact tags to be complete:
```suggestion
[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v1.5.0/pkg/apis/keda/v1alpha1/scaledobject_types.go)
```

**Review by tomkerkhove — COMMENTED** [review]
Thanks for fixing! Made a few suggestions to make it even better

**Comment by Shubham82** [comment]
Thanks @tomkerkhove  for taking look in this. i will do this change.

**Comment by Shubham82** [comment]
Hi @tomkerkhove
I have addressed the comments, PTAL.

File: content/docs/2.1/concepts/scaling-deployments.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ The only constraint is that the target `Custom Resource` must define `/scale` [s
 
 This specification describes the `ScaledObject` Custom Resource definition which is used to define how KEDA should scale your application and what the triggers are. The `.spec.ScaleTargetRef` section holds the reference to the target resource, ie. `Deployment`, `StatefulSet` or `Custom Resource`. 
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/master/api/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/v2.1.0/api/v1alpha1/scaledobject_types.go)
```
Comment (tomkerkhove) [inline content/docs/2.1/concepts/scaling-deployments.md:34]: 2.1 does not exist yet, can you change it to main please?
```suggestion
[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/main/api/v1alpha1/scaledobject_types.go)
```

File: content/docs/2.1/concepts/scaling-jobs.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ For example, if you wanted to use KEDA to run a job for each message that lands
 
 This specification describes the `ScaledJob` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
 
-[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/master/api/v1alpha1/scaledjob_types.go)
+[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/v2.1.0/api/v1alpha1/scaledjob_types.go)
```
Comment (tomkerkhove) [inline content/docs/2.1/concepts/scaling-jobs.md:23]: 2.1 does not exist yet, can you change it to main please?
```suggestion
[`scaledjob_types.go`](https://github.com/kedacore/keda/blob/main/api/v1alpha1/scaledjob_types.go)
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just 2 more changes are required since 2.1 isn't shipped yet

**Comment by Shubham82** [comment]
Sorry @tomkerkhove  for it. I will do it.
Thanks

**Comment by tomkerkhove** [comment]
No problem!

**Comment by Shubham82** [comment]
Hi @tomkerkhove  
Addressed the comments. PTAL.
Thanks.

**Comment by tomkerkhove** [comment]
LGTM, thanks!

**Pull request merged**